rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -60,7 +60,7 @@ Set the value by entering text or by clicking on a checkbox, and complete your e
 +
 
 [[test-configuration-link-globals]]
-Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%]. Only those variables assigned to the phase will be displayed for linking. The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.
+Alternatively, you can set the values of the activity parameters by linking them with a global variable. To do this, click the *Link a global activity parameter* icon image:linkglobalvariable-icon.png[ symbol,1.5%,1.5%]. RPA Manager only shows variables assigned to the test phase and only credentials for which you belong to the xref:processautomation-prepare.adoc#form-create-credential-useraffiliation[affiliated users] for linking. If a credential affiliated to another user is already linked, RPA Manager displays a row of stars to hide the data. The variables are typed. You can only enter or link values of the same type.  The values entered here are only start values and they can be overwritten by other values during the course of the process run.
 * *User Task*:
```

Comment:
I would remove the colons after each bolded list item. I realize this is older text.